packages/circuit-ui/cli/migrate/icons-v2.ts

```diff
@@ -0,0 +1,309 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Transform,
+  JSCodeshift,
+  Collection,
+  JSXAttribute,
+  StringLiteral,
+} from 'jscodeshift';
+
+import { findImportsByPath, findStyledComponentNames } from './utils';
+
+const RENAMED_ICONS = [
+  ['AddItem', 'AddItems', 'Item Catalog'],
```

Comment:
Yes, good idea 👍 It's actually a tiny bit more complicated than that 😅 in this case it's `[oldName: string, newName: string, productName?: string]` (we're warning if product-specific icons are used because eventually they should be "reserved" to the said products)

Reply:
Done in 699fd0c